Makefile

```diff
@@ -2,42 +2,46 @@ HOSTNAME = kentik
 NAMESPACE = automation
 NAME = kentik-cloudexport
 BINARY = terraform-provider-${NAME}
-VERSION := $(shell echo `git tag --list 'v*' | tail -1 | cut -d v -f 2`)
+VERSION := $(shell echo `git tag --list 'v*' | tail -1 | cut -d v -f 2` | sed -e 's/^$$/0.1.0/')
```

Comment:
not sure about the /^$$/ part of sed. Is double $ intended? If it's purpose is to replace blank with 0.1.0, then it should most likely be /^$/

Reply:
I think this is because all `$` characters needs to be doubled in a Makefile, if they should make it to the actual shell command. (see [GNU make manual](https://www.gnu.org/software/make/manual/make.html#Reference))

Reply:
I tested it and it works well